src/main/java/core/basesyntax/Bulldozer.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax;
+
+public class Bulldozer extends Machine {
+    public void doWork() {
+        System.out.println("Bulldozer is now working.");
+    }
+
+    public void stopWork() {
+        System.out.println("Bulldozer has stopped working.");

```

Comment:
Similarly, the message in the 'stopWork' method should be more informative and indicate that it is a Bulldozer that has stopped working.